<?xml version="1.0" encoding="UTF-8" ?>
<testsuites disabled="0" errors="0" failures="8" tests="8" time="74.732">
 <testsuite name="Outbound Preview Dialling Tests" timestamp="2020-04-15T14:54:59" hostname="localhost" time="74.732" errors="0" tests="8" skipped="0" disabled="0" failures="8">
  <testcase classname="Outbound Preview Dialling Tests" name="Agent Login test" time="56.644">
   <failure type="exception" message="Failed: Wait timed out after 30017ms"><![CDATA[TimeoutError: Wait timed out after 30017ms
    at C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2201:17
    at ManagedPromise.invokeCallback_ (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
    at TaskQueue.execute_ (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
    at TaskQueue.executeNext_ (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
    at asyncRun (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27)
    at C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7
    at process._tickCallback (internal/process/next_tick.js:68:7)
From: Task: <anonymous wait>
    at scheduleWait (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2188:20)
    at ControlFlow.wait (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2517:12)
    at thenableWebDriverProxy.wait (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver.js:934:29)
    at run (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\built\browser.js:59:33)
    at ProtractorBrowser.to.(anonymous function) [as wait] (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\built\browser.js:67:16)
    at UserContext.<anonymous> (F:\Automation code\ReleaseAutomation\specs\OutboundPreview.Spec.js:32:17)
    at C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:112:25
    at new ManagedPromise (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1077:7)
    at ControlFlow.promise (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2505:12)
    at schedulerExecute (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:95:18)
From: Task: Run it("Agent Login test") in control flow
    at UserContext.<anonymous> (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:94:19)
    at attempt (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
    at QueueRunner.run (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
    at QueueRunner.execute (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4199:10)
    at Spec.queueRunnerFactory (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:909:35)
    at Spec.execute (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:526:10)
    at UserContext.fn (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:5340:37)
    at attempt (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
    at QueueRunner.run (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
    at QueueRunner.execute (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4199:10)
From asynchronous test: 
Error
    at Suite.<anonymous> (F:\Automation code\ReleaseAutomation\specs\OutboundPreview.Spec.js:23:5)
    at addSpecsToSuite (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1107:25)
    at Env.describe (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1074:7)
    at describe (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4399:18)
    at Object.<anonymous> (F:\Automation code\ReleaseAutomation\specs\OutboundPreview.Spec.js:1:63)
    at Module._compile (internal/modules/cjs/loader.js:688:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:699:10)
    at Module.load (internal/modules/cjs/loader.js:598:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:537:12)]]>
   </failure>
  </testcase>
  <testcase classname="Outbound Preview Dialling Tests" name="Lead Fetch test" time="0.496">
   <failure type="exception" message="Failed: No element found using locator: By(css selector, div[href*=&apos;#/home/phone/pr&apos;])"><![CDATA[NoSuchElementError: No element found using locator: By(css selector, div[href*='#/home/phone/pr'])
    at elementArrayFinder.getWebElements.then (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\built\element.js:814:27)
    at ManagedPromise.invokeCallback_ (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
    at TaskQueue.execute_ (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
    at TaskQueue.executeNext_ (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
    at asyncRun (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27)
    at C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7
    at process._tickCallback (internal/process/next_tick.js:68:7)Error
    at ElementArrayFinder.applyAction_ (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\built\element.js:459:27)
    at ElementArrayFinder.(anonymous function).args [as click] (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\built\element.js:91:29)
    at ElementFinder.(anonymous function).args [as click] (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\built\element.js:831:22)
    at AgentHomePage.ClickPeopleBtn (F:\Automation code\ReleaseAutomation\pages\AgentHomePage.js:19:52)
    at UserContext.<anonymous> (F:\Automation code\ReleaseAutomation\specs\OutboundPreview.Spec.js:37:23)
    at C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:112:25
    at new ManagedPromise (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1077:7)
    at ControlFlow.promise (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2505:12)
    at schedulerExecute (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:95:18)
    at TaskQueue.execute_ (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
From: Task: Run it("Lead Fetch test") in control flow
    at UserContext.<anonymous> (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:94:19)
    at attempt (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
    at QueueRunner.run (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
    at QueueRunner.execute (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4199:10)
    at Spec.queueRunnerFactory (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:909:35)
    at Spec.execute (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:526:10)
    at UserContext.fn (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:5340:37)
    at attempt (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
    at QueueRunner.run (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
    at runNext (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4257:20)
From asynchronous test: 
Error
    at Suite.<anonymous> (F:\Automation code\ReleaseAutomation\specs\OutboundPreview.Spec.js:36:5)
    at addSpecsToSuite (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1107:25)
    at Env.describe (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1074:7)
    at describe (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4399:18)
    at Object.<anonymous> (F:\Automation code\ReleaseAutomation\specs\OutboundPreview.Spec.js:1:63)
    at Module._compile (internal/modules/cjs/loader.js:688:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:699:10)
    at Module.load (internal/modules/cjs/loader.js:598:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:537:12)]]>
   </failure>
  </testcase>
  <testcase classname="Outbound Preview Dialling Tests" name="Admin Login test" time="17.499">
   <failure type="exception" message="Failed: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=81.0.4044.92)
Build info: version: &apos;3.141.59&apos;, revision: &apos;e82be7d358&apos;, time: &apos;2018-11-14T08:25:53&apos;
System info: host: &apos;TAPASWI&apos;, ip: &apos;10.108.10.85&apos;, os.name: &apos;Windows 8.1&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;6.3&apos;, java.version: &apos;9.0.4&apos;
Driver info: driver.version: unknown"><![CDATA[NoSuchWindowError: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=81.0.4044.92)
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:25:53'
System info: host: 'TAPASWI', ip: '10.108.10.85', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '9.0.4'
Driver info: driver.version: unknown
    at Object.checkLegacyResponse (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\error.js:546:15)
    at parseHttpResponse (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\http.js:509:13)
    at doSend.then.response (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\http.js:441:30)
    at process._tickCallback (internal/process/next_tick.js:68:7)
From: Task: WebDriver.findElements(By(xpath, //span[contains(text(), 'Real-Time Outbound')]))
    at thenableWebDriverProxy.schedule (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver.js:807:17)
    at thenableWebDriverProxy.findElements (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver.js:1048:19)
    at ptor.waitForAngular.then (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\built\element.js:159:44)
    at ManagedPromise.invokeCallback_ (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
    at TaskQueue.execute_ (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
    at TaskQueue.executeNext_ (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
    at asyncRun (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27)
    at C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7
From: Task: <anonymous>
    at Timeout.pollCondition [as _onTimeout] (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2195:19)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
From: Task: <anonymous wait>
    at scheduleWait (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2188:20)
    at ControlFlow.wait (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2517:12)
    at thenableWebDriverProxy.wait (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver.js:934:29)
    at run (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\built\browser.js:59:33)
    at ProtractorBrowser.to.(anonymous function) [as wait] (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\built\browser.js:67:16)
    at UserContext.<anonymous> (F:\Automation code\ReleaseAutomation\specs\OutboundPreview.Spec.js:67:17)
    at C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:112:25
    at new ManagedPromise (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1077:7)
    at ControlFlow.promise (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2505:12)
    at schedulerExecute (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:95:18)
From: Task: Run it("Admin Login test") in control flow
    at UserContext.<anonymous> (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:94:19)
    at attempt (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
    at QueueRunner.run (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
    at QueueRunner.execute (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4199:10)
    at Spec.queueRunnerFactory (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:909:35)
    at Spec.execute (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:526:10)
    at UserContext.fn (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:5340:37)
    at attempt (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
    at QueueRunner.run (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
    at runNext (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4257:20)
From asynchronous test: 
Error
    at Suite.<anonymous> (F:\Automation code\ReleaseAutomation\specs\OutboundPreview.Spec.js:54:5)
    at addSpecsToSuite (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1107:25)
    at Env.describe (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1074:7)
    at describe (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4399:18)
    at Object.<anonymous> (F:\Automation code\ReleaseAutomation\specs\OutboundPreview.Spec.js:1:63)
    at Module._compile (internal/modules/cjs/loader.js:688:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:699:10)
    at Module.load (internal/modules/cjs/loader.js:598:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:537:12)]]>
   </failure>
  </testcase>
  <testcase classname="Outbound Preview Dialling Tests" name="Dashboard Presence test" time="0.022">
   <failure type="exception" message="NoSuchWindowError: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=81.0.4044.92)
Build info: version: &apos;3.141.59&apos;, revision: &apos;e82be7d358&apos;, time: &apos;2018-11-14T08:25:53&apos;
System info: host: &apos;TAPASWI&apos;, ip: &apos;10.108.10.85&apos;, os.name: &apos;Windows 8.1&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;6.3&apos;, java.version: &apos;9.0.4&apos;
Driver info: driver.version: unknown"><![CDATA[NoSuchWindowError: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=81.0.4044.92)
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:25:53'
System info: host: 'TAPASWI', ip: '10.108.10.85', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '9.0.4'
Driver info: driver.version: unknown
    at Object.checkLegacyResponse (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\error.js:546:15)
    at parseHttpResponse (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\http.js:509:13)
    at doSend.then.response (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\http.js:441:30)
    at process._tickCallback (internal/process/next_tick.js:68:7)
From: Task: WebDriver.takeScreenshot()
    at thenableWebDriverProxy.schedule (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver.js:807:17)
    at thenableWebDriverProxy.takeScreenshot (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver.js:1085:17)
    at run (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\built\browser.js:59:33)
    at ProtractorBrowser.to.(anonymous function) [as takeScreenshot] (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\built\browser.js:67:16)
    at F:\Automation code\ReleaseAutomation\conf.js:58:21
    at ManagedPromise.invokeCallback_ (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
    at TaskQueue.execute_ (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
    at TaskQueue.executeNext_ (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
    at asyncRun (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27)
    at C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7]]>
   </failure>
  </testcase>
  <testcase classname="Outbound Preview Dialling Tests" name="Outbound Telemetry Test" time="0.027">
   <failure type="exception" message="NoSuchWindowError: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=81.0.4044.92)
Build info: version: &apos;3.141.59&apos;, revision: &apos;e82be7d358&apos;, time: &apos;2018-11-14T08:25:53&apos;
System info: host: &apos;TAPASWI&apos;, ip: &apos;10.108.10.85&apos;, os.name: &apos;Windows 8.1&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;6.3&apos;, java.version: &apos;9.0.4&apos;
Driver info: driver.version: unknown"><![CDATA[NoSuchWindowError: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=81.0.4044.92)
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:25:53'
System info: host: 'TAPASWI', ip: '10.108.10.85', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '9.0.4'
Driver info: driver.version: unknown
    at Object.checkLegacyResponse (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\error.js:546:15)
    at parseHttpResponse (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\http.js:509:13)
    at doSend.then.response (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\http.js:441:30)
    at process._tickCallback (internal/process/next_tick.js:68:7)
From: Task: WebDriver.takeScreenshot()
    at thenableWebDriverProxy.schedule (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver.js:807:17)
    at thenableWebDriverProxy.takeScreenshot (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver.js:1085:17)
    at run (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\built\browser.js:59:33)
    at ProtractorBrowser.to.(anonymous function) [as takeScreenshot] (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\built\browser.js:67:16)
    at F:\Automation code\ReleaseAutomation\conf.js:58:21
    at ManagedPromise.invokeCallback_ (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
    at TaskQueue.execute_ (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
    at TaskQueue.executeNext_ (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
    at asyncRun (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27)
    at C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7]]>
   </failure>
  </testcase>
  <testcase classname="Outbound Preview Dialling Tests" name="Admin Logout Test" time="0.01">
   <failure type="exception" message="NoSuchWindowError: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=81.0.4044.92)
Build info: version: &apos;3.141.59&apos;, revision: &apos;e82be7d358&apos;, time: &apos;2018-11-14T08:25:53&apos;
System info: host: &apos;TAPASWI&apos;, ip: &apos;10.108.10.85&apos;, os.name: &apos;Windows 8.1&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;6.3&apos;, java.version: &apos;9.0.4&apos;
Driver info: driver.version: unknown"><![CDATA[NoSuchWindowError: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=81.0.4044.92)
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:25:53'
System info: host: 'TAPASWI', ip: '10.108.10.85', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '9.0.4'
Driver info: driver.version: unknown
    at Object.checkLegacyResponse (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\error.js:546:15)
    at parseHttpResponse (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\http.js:509:13)
    at doSend.then.response (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\http.js:441:30)
    at process._tickCallback (internal/process/next_tick.js:68:7)
From: Task: WebDriver.takeScreenshot()
    at thenableWebDriverProxy.schedule (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver.js:807:17)
    at thenableWebDriverProxy.takeScreenshot (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver.js:1085:17)
    at run (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\built\browser.js:59:33)
    at ProtractorBrowser.to.(anonymous function) [as takeScreenshot] (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\built\browser.js:67:16)
    at F:\Automation code\ReleaseAutomation\conf.js:58:21
    at ManagedPromise.invokeCallback_ (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
    at TaskQueue.execute_ (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
    at TaskQueue.executeNext_ (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
    at asyncRun (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27)
    at C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7]]>
   </failure>
  </testcase>
  <testcase classname="Outbound Preview Dialling Tests" name="Disposition Test" time="0.01">
   <failure type="exception" message="NoSuchWindowError: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=81.0.4044.92)
Build info: version: &apos;3.141.59&apos;, revision: &apos;e82be7d358&apos;, time: &apos;2018-11-14T08:25:53&apos;
System info: host: &apos;TAPASWI&apos;, ip: &apos;10.108.10.85&apos;, os.name: &apos;Windows 8.1&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;6.3&apos;, java.version: &apos;9.0.4&apos;
Driver info: driver.version: unknown"><![CDATA[NoSuchWindowError: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=81.0.4044.92)
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:25:53'
System info: host: 'TAPASWI', ip: '10.108.10.85', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '9.0.4'
Driver info: driver.version: unknown
    at Object.checkLegacyResponse (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\error.js:546:15)
    at parseHttpResponse (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\http.js:509:13)
    at doSend.then.response (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\http.js:441:30)
    at process._tickCallback (internal/process/next_tick.js:68:7)
From: Task: WebDriver.takeScreenshot()
    at thenableWebDriverProxy.schedule (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver.js:807:17)
    at thenableWebDriverProxy.takeScreenshot (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver.js:1085:17)
    at run (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\built\browser.js:59:33)
    at ProtractorBrowser.to.(anonymous function) [as takeScreenshot] (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\built\browser.js:67:16)
    at F:\Automation code\ReleaseAutomation\conf.js:58:21
    at ManagedPromise.invokeCallback_ (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
    at TaskQueue.execute_ (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
    at TaskQueue.executeNext_ (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
    at asyncRun (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27)
    at C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7]]>
   </failure>
  </testcase>
  <testcase classname="Outbound Preview Dialling Tests" name="Second OutBoundPreview dial test" time="0.009">
   <failure type="exception" message="NoSuchWindowError: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=81.0.4044.92)
Build info: version: &apos;3.141.59&apos;, revision: &apos;e82be7d358&apos;, time: &apos;2018-11-14T08:25:53&apos;
System info: host: &apos;TAPASWI&apos;, ip: &apos;10.108.10.85&apos;, os.name: &apos;Windows 8.1&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;6.3&apos;, java.version: &apos;9.0.4&apos;
Driver info: driver.version: unknown"><![CDATA[NoSuchWindowError: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=81.0.4044.92)
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:25:53'
System info: host: 'TAPASWI', ip: '10.108.10.85', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '9.0.4'
Driver info: driver.version: unknown
    at Object.checkLegacyResponse (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\error.js:546:15)
    at parseHttpResponse (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\http.js:509:13)
    at doSend.then.response (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\http.js:441:30)
    at process._tickCallback (internal/process/next_tick.js:68:7)
From: Task: WebDriver.takeScreenshot()
    at thenableWebDriverProxy.schedule (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver.js:807:17)
    at thenableWebDriverProxy.takeScreenshot (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver.js:1085:17)
    at run (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\built\browser.js:59:33)
    at ProtractorBrowser.to.(anonymous function) [as takeScreenshot] (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\built\browser.js:67:16)
    at F:\Automation code\ReleaseAutomation\conf.js:58:21
    at ManagedPromise.invokeCallback_ (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
    at TaskQueue.execute_ (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
    at TaskQueue.executeNext_ (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
    at asyncRun (C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27)
    at C:\Users\Welcome\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7]]>
   </failure>
  </testcase>
 </testsuite>
</testsuites>